app/src/components/Home/components/ChangelogCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Col, Row, Typography, Skeleton } from "antd";
+import { ChangeLogType, Changelog } from "components/Home/types";
+import Logger from "lib/logger";
+import "./changelogCard.scss";
+
+export const ChangeLogCard: React.FC = () => {
+  const [logs, setLogs] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch("https://requestly.tech/api/mockv2/v1/rq-changelog?teamId=LmyapmzY39XTFXvua6eX")
```

Comment:
How we are going to maintain the changelog?

Reply:
I have created a mock API in RQ official workspace and added a static JSON response. We just need to update the fields.